src/ReachSets/DiscretePost/DecomposedDiscretePost.jl

```diff
@@ -0,0 +1,144 @@
+export DecomposedDiscretePost
+
+"""
+    DecomposedDiscretePost <: DiscretePost
+
+Textbook implementation of a discrete post operator, but with lazy decomposed intersections.
+
+### Fields
+
+- `options` -- an `Options` structure that holds the algorithm-specific options
+
+### Algorithm
+
+The algorithm is based on [Flowpipe-Guard Intersection for Reachability
+Computations with Support Functions](http://spaceex.imag.fr/sites/default/files/frehser_adhs2012.pdf).
```

Comment:
I would rather give a link to our arXiv version.

Also please add a reference to the `BFFPS19` CPost as in
```
This discrete-post operator can only be used in combination with the
continuous-post operator [`BFFPS19`](@ref).
```